packages/docs/getting-started/running-a-validator.md

```diff
@@ -271,10 +273,16 @@ You can see all the options using the following command:
 ./run-docker-validator-network.sh help
 ```
 
+## Get elected as validator
+
 ### Obtain and lock up some Celo Gold for staking
 
 To participate in The Great Celo Stake Off (aka TGCSO) and get fauceted it's necessary to register online via an [online form](https://docs.google.com/forms/d/e/1FAIpQLSfbn5hTJ4UIWpN92-o2qMTUB0UnrFsL0fm97XqGe4VhhN_r5A/viewform).
 
+### Submitting transactions
+
+We created the `celocli` to allow users to easily interact with the smart contracts on the command line. The `celocli` generally expects a celo node, by default we are using the RPC API interface at `http://localhost:8545`. To submit transactions, the node should have the account unlocked (the one that is usually specified with `--from`). You can specify that through the `--unlock` parameter when starting your celo node. Otherwise you can use `celocli` to unlock the account with `celocli account:unlock` (assuming you have the `personal` module enabled).
```

Comment:
the note on `personal` module being enabled feels like it could be confusing to someone who's not familiar with ethereum RPCs. Maybe be more specific by saying it's an RPC API

Reply:
I'm going to change a whole lot about key management, so maybe stay tuned?